_config.yml

```diff
@@ -4,7 +4,7 @@
 # Make our URLs look nice. We're not a blog with dated posts, we just want the title.
 permalink:     /:title.html
 
-gems:
+plugins:
```

Comment:
looks more like something wasn't right when I checked in the final v2.5.0 doc changes (e.g. moving "latest" link from 2.4 to 2.5).